azure-pipelines.yml

```diff
@@ -190,6 +190,8 @@ stages:
           matrix:
             IT_es91x:
               ROR_TASK: integration_es91x
+            IT_es90x:
```

Comment:
At the moment, for test purposes, I configured it to run Windows tests for a few representative modules, and only newest one on Linux.

I propose to always run Windows tests on the newest ES version - even in PR's. It is only one job more, and will allow to catch errors and incompatibilities before merge with develop, WDYT about that?. Which versions should be tested on develop and master pipelines?